scripts/http_client_notification.py

```diff
@@ -0,0 +1,42 @@
+import requests
+import json
+from loguru import logger
+from websocket_client import get_jwt_access, FORUM_HOST, FORUM_PORT
+
+
+def get_notifications(access_token):
+    response = requests.get(
+        f"http://{FORUM_HOST}:{FORUM_PORT}/notifications/",
+        headers={"Authorization": f"Bearer {access_token}"},
+        cookies={"access_token": access_token}
+    )
+    if response.status_code == 200:
+        logger.info("Notifications:")
+        logger.info(json.dumps(response.json(), indent=4))
+        return response.json()
+    else:
+        logger.error(f"Failed to get notifications: {response.status_code}")
+
+
+def mark_notification_as_read(access_token, notification_id):
+    response = requests.put(
+        f"http://{FORUM_HOST}:{FORUM_PORT}/notifications/{notification_id}/",
+        headers={"Authorization": f"Bearer {access_token}"},
+        cookies={"access_token": access_token}
+    )
+    if response.status_code == 200:
+        logger.info("Notification marked as read:")
+        logger.info(json.dumps(response.json(), indent=4))
+    else:
+        logger.error(f"Failed to mark notification as read: {response.status_code}")
+
+
+if __name__ == "__main__":
+    access_token = get_jwt_access()
+
+    notifications = get_notifications(access_token)
+
+    if notifications:
+        first_notification_id = notifications[0]["id"]
```

Comment:
The code assumes that the notifications list will always have at least one item. And what if empty list?

Reply:
If we don't receive a notifications for the current authenticated user `if notifications:`, the code doesn't mark the first notification as read (doesn't remove the current user from the recipients). This code is for testing purposes only, we could even select which notification we want to read (pass its notification_id), but I decided to do it automatically.